src/Card/CardHeader.jsx

```diff
@@ -26,18 +27,26 @@ export default class CardHeader extends Component {
       compose: 'Merge',
     });
 
+    if (typeof title === 'string') {
+      return (
+        <div className={this.createHeaderClassName()}>
+          {actions ? <CardActions {...this.props} /> : null }
+          {title ? <div className={themeStyles.titleWrapper}><span className={themeStyles.title} title={title}>{title}</span></div> : null }
+        </div>
+      );
+    }
     return (
-      <div className={this.createHeaderClassName()}>
-        {actions ? <CardActions {...this.props} /> : null }
-        {title ? <div className={themeStyles.titleWrapper}><span className={themeStyles.title} title={title}>{title}</span></div> : null }
-      </div>
+      <PlaceholderTextComponent {...this.props} />
     );
   }
 }
 
 CardHeader.propTypes = {
   cardId: PropTypes.string.isRequired,
-  title: PropTypes.string,
+  title: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.instanceOf(Object),
```

Comment:
we should be explicit here ... instead of instanceOf(Object), this should be PropTypes.instanceOf(PlaceholderComponent)